libcodechecker/server/client_db_access_handler.py

```diff
@@ -427,7 +473,7 @@ def __get_run_ids_to_query(self, session, cmp_data=None):
 
     @timeit
     def getRunData(self, run_name_filter):
-
+        self.__require_access()
```

Comment:
Permissions are read from the config database for each api call?

Reply:
Yes. This is a security measure. It is not a good option security-wise to *ever* cache any sort of "permission" descriptor.